buildpacks/jvm-function-invoker/src/layers/runtime.rs

```diff
@@ -0,0 +1,107 @@
+use std::path::{Path, PathBuf};
+
+use libcnb::data::layer_content_metadata::LayerContentMetadata;
+use libcnb::layer_lifecycle::{LayerLifecycle, ValidateResult};
+use serde::{Deserialize, Serialize};
+use thiserror::Error;
+
+use crate::error::JvmFunctionInvokerBuildpackError;
+use crate::JvmFunctionInvokerBuildpackMetadata;
+use libcnb::{BuildContext, GenericPlatform};
+use libherokubuildpack::{log_header, log_info, DownloadError, download_file, sha256};
```

Comment:
`cargo fmt -- --check --verbose` says:

```suggestion
use libherokubuildpack::{download_file, log_header, log_info, sha256, DownloadError};
```